docs/references/nextjs/authentication-object.mdx

```diff
@@ -5,6 +5,8 @@ description: The Authentication object contains information about the current us
 
 # `Authentication` Object
 
+Both [`auth()`](/docs/references/nextjs/auth) and  [`getAuth()`](/docs/references/nextjs/get-auth) return an Authentication object. This JavaScript object contains important information like session data, your user's ID, as well as their org ID.
+
 ## Properties
 
 | Name | Description |
```

Comment:
Not sure why I can't select lines in the table, but the 'orgId', 'orgRole', 'orgSli' entries should all change to say 'user's active organization'